wbstack/src/Settings/LocalSettings.php

```diff
@@ -525,13 +525,26 @@ function onPersonalUrlsConfirmAccount( array &$personal_urls, Title $title, Skin
     }
 }
 
-# CirrusSearch etc.
-// wfLoadExtension( 'Elastica' );
-// wfLoadExtension( 'CirrusSearch' );
-// wfLoadExtension( 'WikibaseCirrusSearch' );
-// if ( $wikiInfo->getSetting('wwExtEnableWikibaseLexeme') ) {
-//     wfLoadExtension('WikibaseLexemeCirrusSearch');
-// }
+// ElasticSearch
+if ( $wikiInfo->getSetting( 'wwExtEnableElasticSearch' ) ) {
+    error_log( "elastica enabled" );
+    wfLoadExtension( 'Elastica' );
+    wfLoadExtension( 'CirrusSearch' );
+    wfLoadExtension( 'WikibaseCirrusSearch' );
+
+    // If Wikibase Lexemes are enabled, enable lexeme cirrus search
+    if ( $wikiInfo->getSetting('wwExtEnableWikibaseLexeme') ) {
+        wfLoadExtension('WikibaseLexemeCirrusSearch');
+    }
+
+    // prepends indices with subdomain
+    $wgCirrusSearchIndexBaseName = $wikiInfo->requestDomain;
+
+    $wgSearchType = 'CirrusSearch';
+    $wgCirrusSearchServers = [ getenv('MW_ELASTICSEARCH_HOST') ];
```

Comment:
I think we might wanna do something like this instead since this would be deployed in k8s. Not sure how the formatting for that should look for the env var though

```
    $wgCirrusSearchDefaultCluster = 'default';
    $wgCirrusSearchClusters = [
        'default' => [
            [ 'host' => '172.17.0.1', 'port' => 8234 ],
        ]
    ];
```

Reply:
done